src/app/screens/sendBrc20OneStep/brc20TransferForm.tsx

```diff
@@ -193,10 +196,9 @@ function Brc20TransferForm(props: Props) {
               />
             </InputFieldContainer>
           </AmountInputContainer>
-          <ErrorContainer>
-            <ErrorText>{recipientError}</ErrorText>
-          </ErrorContainer>
+          <ErrorContainer>{recipientError && <InputFeedback {...recipientError} />}</ErrorContainer>
         </InputGroup>
+        <StyledCallout bodyText={t('MAKE_SURE_THE_RECIPIENT')} />
       </div>
       <NextButtonContainer>
```

Comment:
From the screenshot, something looks funky with the next button container. Should it not have a margin at the top above the button?

Reply:
Also, it's transparent 🙈 

<img width="346" alt="image" src="https://github.com/secretkeylabs/xverse-web-extension/assets/3322667/b02155d3-7166-4871-b4cc-d8781ddf50fc">

Reply:
nice one. fixed here:

![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/5796d8f7-f563-43ff-afe1-81f8f82a1189)